CHANGES

```diff
@@ -42,6 +42,9 @@ Features added
   text
 * #9176: i18n: Emit a debug message if message catalog file not found under
   :confval:`locale_dirs`
+* #9016: linkcheck: failed to check the anchor of github.com
+* #9016: linkcheck: Add a new event :event:`linkcheck-process-uri` to modify
+  URIs before checking hyperlinks
```

Comment:
There will be two bullet points for this entry.

```suggestion
* #9016: linkcheck: failed to check the anchor of github.com
  Add a new event :event:`linkcheck-process-uri` to modify URIs before checking hyperlinks
```

Reply:
I think it's not so wrong to add two entries to CHANGES per issue. But first entry is not good because it does not describe a "new feature". This entry is like a bug fix.